sdk/src/test/scala/io/horizen/utxo/storage/SidechainHistoryStorageTest.scala

```diff
@@ -673,13 +673,13 @@ class SidechainHistoryStorageTest extends JUnitSuite with MockitoSugar with Side
   def testChainWithoutMcBlockHeaders(): Unit = {
 
     // build a chain with genesis block + 10 sc blocks with no mc references but the 4th block
-    // the history storage and all mocked data have params.maxHistoryRewritingLength=5
+    // the history storage and all mocked data have params.maxHistoryRewritingLength=6
     //   G <- B1 <- B2 <-.. <-B4 <- .. <-B10
     //   ^                    ^
     //   |                    |
     //  mcRef                mcRef
 
-    val t : (SidechainHistoryStorage, Seq[SidechainBlock]) = getTestChainData(spanLength = 10, mcHeaderPos = 4, inMaxHistoryRewritingLength = 5)
+    val t : (SidechainHistoryStorage, Seq[SidechainBlock]) = getTestChainData(spanLength = 10, mcHeaderPos = 4, inMaxHistoryRewritingLength = 6)
```

Comment:
As an alternative, you could write like this:
`val (historyStorage: SidechainHistoryStorage, chainList: Seq[SidechainBlock]) = getTestChainData(spanLength = 10, mcHeaderPos = 4, inMaxHistoryRewritingLength = 6)`
It's maybe a little bit easier to read and it saves 2 lines of codes.

Reply:
done